projects/pt1/python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -655,6 +656,39 @@ def TransposeIntNegDimsModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class PixelShuffleModuleStatic_3_18_2_2(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([None, ([3, 18, 2, 2], torch.float32, True)])
+    def forward(self, x):
+        return torch.ops.aten.pixel_shuffle(x, 3)
+
+@register_test_case(module_factory=lambda: PixelShuffleModuleStatic_3_18_2_2())
+def PixelShuffleModuleStatic_3_18_2_2_basic(module, tu: TestUtils):
```

Comment:
nit: no need to write the shape in the test name. I think the important bit here is the input dtype, so I would call the tests something like `PixelShuffleModuleStaticInt64` and `PixelShuffleModuleStaticFloat32`

Reply:
Initially my motivation for the second test was for the rank-3 case (no leading dimensions). Making it a different dtype was just an extra, to get coverage in a different direction.  I've gone with names `Rank4Float32` and `Rank3Int64`

Reply:
Makes sense, thanks!